source/_integrations/satel_integra.markdown

```diff
@@ -71,9 +71,9 @@ partitions:
       required: true
       type: string    
     arm_home_mode:
-      description: The mode in which the partition is armed when 'arm home' is used. Possible options are `1`,`2` or `3`. For more information on what the differences are between them, please refer to Satel Integra manual.
+      description: The mode in which the partition is armed when 'arm home' is used. Possible options are `0`,`1`,`2` or `3`. For more information on what the differences are between them, please refer to Satel Integra manual.
```

Comment:
0 is not a valid value:

![CleanShot 2023-11-07 at 23 25 49@2x](https://github.com/home-assistant/home-assistant.io/assets/195327/d70fe654-3d50-42a9-9d4d-e38e06ebe76f)

And the default is 0:

![CleanShot 2023-11-07 at 23 26 12@2x](https://github.com/home-assistant/home-assistant.io/assets/195327/7a3bf3a2-63d4-4d9f-91ac-e08f9823876e)

Reply:
I'm new here so my apologies for the misunderstanding when creating a pull request, I was only trying to show the possibility of setting a 0 arm mode (which is in fact default and what you would normally use for daily usage). So I guess this is more a suggestion to modify the codebase than anything else.
Also see https://github.com/home-assistant/home-assistant.io/issues/22588 where this has been raised before.
Hope you don't mind me commenting here, if there is a better way to get involved without knowing about coding myself please let me know as I have worked as a distributor for Satel for 6 years now and I think I can contribute.

Reply:
>  I was only trying to show the possibility of setting a 0 arm mode

As per above, once can't. The configuration would be considered invalid for Home Assistant.

../Frenck